sql/core/src/main/scala/org/apache/spark/sql/execution/SQLExecution.scala

```diff
@@ -71,13 +74,23 @@ object SQLExecution {
       // streaming queries would give us call site like "run at <unknown>:0"
       val callSite = sc.getCallSite()
 
+      val truncateLength = sparkSession.sparkContext.conf.get(SQL_EVENT_TRUNCATE_LENGTH)
+
+      val desc = Option(sparkSession.sparkContext
+        .getLocalProperty(SparkContext.SPARK_JOB_DESCRIPTION))
+        .filter(_ => truncateLength > 0)
+        .map { sqlStr =>
+          Utils.redact(sparkSession.sessionState.conf.stringRedactionPattern, sqlStr)
```

Comment:
ditto

Reply:
this is SQLConf got via sessionstate.. i do not see any spark context